### PR TITLE
Postgres cost metric update

### PR DESCRIPTION
**Pull request by chrislarson1**

### CODE REVIEW

File: postgres/experiment.yaml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 apiVersion: redskyops.dev/v1alpha1
 kind: Experiment
 metadata:
-  name: postgres-example
+  name: postgres-example-test-cost-metric
```
Comment (bradbeam) [inline postgres/experiment.yaml:4]: probably want to leave this at the example name

File: postgres/experiment.yaml [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 spec:
   optimization:
   - name: "experimentBudget"
-    value: "40" #number of trials
+    value: "100" #number of trials
```
Comment (bradbeam) [inline postgres/experiment.yaml:8]: this was set to 40 because we usually recommend 20*parameters

File: postgres/experiment.yaml (hunk 1 above)
Comment (chrislarson1) [reply postgres/experiment.yaml:4]: Nice catch.

File: postgres/experiment.yaml (hunk 2 above)
Comment (chrislarson1) [reply postgres/experiment.yaml:8]: Nice catch.

File: postgres/experiment.yaml [hunk 3]
```diff
@@ -21,7 +21,13 @@ spec:
...
   - name: cost
     minimize: true
-    query: "{{div (add (mul .Values.cpu 22) (mul .Values.memory 3)) 1000}}"
+    type: pods
+    # Note that these cost weights are specific to GKE and represent $22/month/cpu and $3/month/GB
+    query: '{{resourceRequests .Pods "cpu=0.022,memory=0.000000000003"}}'
+    selector:
+      matchLabels:
+        component: postgres
+    # query: "{{div (add (mul .Values.cpu 22) (mul .Values.memory 3)) 1000}}"
```
Comment (bradbeam) [inline postgres/experiment.yaml:30]: let's drop this commented line and we should be good to go

**Review by bradbeam — APPROVED** [review]

**Pull request merged**